demo/image_mod/run.py

```diff
@@ -1,12 +1,16 @@
 import gradio as gr
+import os
 
 
 def image_mod(image):
     return image.rotate(45)
 
 
 demo = gr.Interface(image_mod, gr.inputs.Image(type="pil"), "image", 
-    flagging_options=["blurry", "incorrect", "other"])
+    flagging_options=["blurry", "incorrect", "other"], examples=[
+        os.path.join(os.path.dirname(__file__), "images/cheetah1.jpg"),
```

Comment:
is `os.path.join(os.path.dirname())`  necessary? cant you just do `images/cheetah1.jpg`.

Reply:
This allows the demo (run.py) file to be run from anywhere